docs/src/browsers.md

```diff
@@ -542,7 +542,7 @@ set NODE_EXTRA_CA_CERTS="C:\certs\root.crt"
 $Env:NODE_EXTRA_CA_CERTS="C:\certs\root.crt"
 ```
 
-If your network is slow to connect to Playwright browser archive, you can increase the connection timeout in milliseconds with `PLAYWRIGHT_DOWNLOAD_CONNECTION_TIMEOUT` environment variable:
+If your network is slow to connect to Playwright browser archive, you can increase the overall download timeout in milliseconds with `PLAYWRIGHT_DOWNLOAD_CONNECTION_TIMEOUT` environment variable:
```

Comment:
```suggestion
If your network is slow to connect to Playwright browser archive, you can increase the connection and idle timeout in milliseconds with `PLAYWRIGHT_DOWNLOAD_CONNECTION_TIMEOUT` environment variable:
```

Its not the overall timeout, more the max of connection timeout and idle timeout.

Can be tested via `PLAYWRIGHT_DOWNLOAD_CONNECTION_TIMEOUT=1000 npx playwright install chromium` which works for me while the whole download takes around 5 seconds.

Reply:
Could you please test it on a mac? For me on macOS Ventura 13.5 with node18 it cancelled an established connection at 8%.

Reply:
Works for me with Node.js 18.18.2 and macOS 14.2. We have CDN fallbacks in place, fails indeed for the first CDN but passes with akamai.

Reply:
You are right... I'm sorry... I saw it a few times before, so I've removed the timeout and I didn't test again.

I tried it now, and it mostly works as you wrote (with the 1000ms)... but with a shorter timeout (100ms) I do see this (sometimes - not always)... so maybe it is some very specific timing issue, when the timeout (cancellation) is triggered, but it does start before it is fully cancelled. Not sure...

```sh
➜  autotests git:(feat/tests) ✗ PLAYWRIGHT_DOWNLOAD_HOST=https://corp-artifactory/playwright-azureedge-net PLAYWRIGHT_DOWNLOAD_CONNECTION_TIMEOUT=100 NO_PROXY=* npx playwright install chromium
Downloading Chromium 120.0.6099.28 (playwright build v1091) from https://corp-artifactory/playwright-azureedge-net/builds/chromium/1091/chromium-mac-arm64.zip
131 Mb [==                  ] 8% 6.4sError: Request to https://corp-artifactory/playwright-azureedge-net/builds/chromium/1091/chromium-mac-arm64.zip timed out after 100ms
    at ClientRequest.rejectOnTimeout (/Users/me/Projects/autotests/node_modules/playwright-core/lib/utils/network.js:78:15)
    at Object.onceWrapper (node:events:631:28)
    at ClientRequest.emit (node:events:517:28)
    at TLSSocket.emitRequestTimeout (node:_http_client:847:9)
    at Object.onceWrapper (node:events:631:28)
    at TLSSocket.emit (node:events:529:35)
    at Socket._onTimeout (node:net:598:8)
    at listOnTimeout (node:internal/timers:569:17)
    at process.processTimers (node:internal/timers:512:7)
Failed to install browsers
Error: Failed to download Chromium 120.0.6099.28 (playwright build v1091), caused by
Error: Download failure, code=1
    at ChildProcess.<anonymous> (/Users/me/Projects/autotests/node_modules/playwright-core/lib/server/registry/browserFetcher.js:92:16)
    at ChildProcess.emit (node:events:517:28)
    at ChildProcess._handle.onexit (node:internal/child_process:292:12)
```